tests/common.go

```diff
@@ -2829,6 +2829,15 @@ func ValidateClusterSize(count int64) {
 	currentNodeCount, err := Inst().S.GetASGClusterSize()
 	log.FailOnError(err, "Failed to Get ASG Cluster Size")
 
+	if Inst().S.String() == openshift.SchedName {
+		isPxOnMaster, err := IsPxRunningOnMaster()
+		log.FailOnError(err, "Failed to check if px is running on master")
+		if !isPxOnMaster {
+			//Removing master nodes for currentNodeCount
+			currentNodeCount = currentNodeCount - 3
```

Comment:
This assumes 3 masters always. Can we have a count of number of masters and deduct that instead.

Reply:
Fixed